src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,48 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.bp-ColorPickerPalette {
+    position: absolute;
+    top: -55px;
```

Comment:
How is this value derived? Can we use a relative value, instead?

Reply:
Yep, I can do a combination of a few other element values.

Reply:
We should try to avoid using any absolute values, if possible. Since this is absolutely-positioned, we should be able to use a relative (percentage) value along with a transform, similar to the `right` calculation.

Reply:
Otherwise, if the goal is to offset the element above its parent, can we use `bottom: 0` and allow the arrow to push the palette up as needed?

Reply:
Would using `bottom: 135%` suffice? It looks like that achieves the desired result without the usage of a transform.
![Screen Shot 2020-12-03 at 11 18 28 AM](https://user-images.githubusercontent.com/7213887/101077230-69e66380-3559-11eb-80c7-3c0e5b476f0a.png)